suite/auto-sync/Update-Arch.sh

```diff
@@ -0,0 +1,279 @@
+#!/bin/sh
+
+pause() {
+  echo "Press any key to exit"
+  read -r
```

Comment:
In bash this doesn't wait for the user input. It needs a variable:
`read -r input`

Reply:
What should I use to wait for input, what I had before was nonstandard bash extentions.

Reply:
Doesn't matter. Only the argument must be set.

```suggestion
  read -r dummy_var
```